### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by arihant2math**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

I added more aliases.

### CODE REVIEW

File: topics/python/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-aliases: python3, python2, python-3, python-2, python27, python-3-6, python-3-5, py
+aliases: python3, python2, python-3, python-2, python39, python38, python37, python36, python35, python28, python27, python-39, python-38, python-37, python-36, python-35, python-28, python-27, py
```
Comment (nholden) [inline topics/python/index.md:2]: Many public repositories use the `python-3-6` topic, but none use the `python-36` topic. What's the rationale behind this change?

**Review by nholden — COMMENTED** [review]
Thanks for this contribution, @arihant2math!

File: topics/python/index.md (hunk 1 above)
Comment (arihant2math) [reply topics/python/index.md:2]: To standardize it, just in case.

Comment (nholden) [reply topics/python/index.md:2]: The community has standardized on the `python-3-6` format, not `python-36`. There are 895 repositories with [the `python-3-6` topic](https://github.com/topics/python-3-6) and zero with [the `python-36` topic](https://github.com/topics/python-36).

It doesn't seem valuable to remove aliases for topics that people are using or add aliases for topics that people are not using, so I'm going to close this PR out. Let me know if I'm misunderstanding, and I'd be open to discussing it further!
